client/src/pages/ItemsList.js

```diff
@@ -1,240 +1,213 @@
 import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
-import { connect } from 'react-redux';
-import { bindActionCreators } from 'redux';
 import { useTable } from 'react-table';
-import * as actions from '../actions';
 import { DeleteButton } from '../components/buttons';
+import api from '../api';
 
-import MaUTable from '@material-ui/core/Table'
-import {
-    CssBaseline,
-    TableBody,
-    TableCell,
-    TableHead,
-    TableRow,
-} from '@material-ui/core';
+import MaUTable from '@material-ui/core/Table';
+import { CssBaseline, TableBody, TableCell, TableHead, TableRow } from '@material-ui/core';
 
 import styled from 'styled-components';
 
 const Wrapper = styled.div`
-    padding: 0 40px 40px 40px;
+  padding: 0 40px 40px 40px;
 
-    @media screen and (max-width: 420px) {
-        padding-left: 0.5em;
-        padding-right: 0.5em;
-    }
+  @media screen and (max-width: 420px) {
+    padding-left: 0.5em;
+    padding-right: 0.5em;
+  }
 `;
 
 const Table = ({ columns, data }) => {
-    const {
-        getTableProps,
-        headerGroups,
-        rows,
-        prepareRow
-    } = useTable({
-      columns,
-      data
-    });
-
-    return (
-        <MaUTable {...getTableProps()}>
-            <TableHead>
-                {headerGroups.map(headerGroup => (
-                    <TableRow {...headerGroup.getHeaderGroupProps()}>
-                        {headerGroup.headers.map(column => (
-                            <TableCell {...column.getHeaderProps()}>
-                                {column.render('Header')}
-                            </TableCell>
-                        ))}
-                    </TableRow>
-                ))}
-            </TableHead>
-            <TableBody>
-                {rows.map((row, i) => {
-                    prepareRow(row)
-                    return (
-                        <TableRow
-                            data-row-item-id={row.values._id}
-                            {...row.getRowProps()}
-                        >
-                            {row.cells.map(cell => {
-                                return (
-                                    <TableCell {...cell.getCellProps()}>
-                                        {cell.render('Cell')}
-                                    </TableCell>
-                                )
-                            })}
-                        </TableRow>
-                    )
-                })}
-            </TableBody>
-        </MaUTable>
-    )
+  const { getTableProps, headerGroups, rows, prepareRow } = useTable({
+    columns,
+    data,
+  });
+
+  return (
+    <MaUTable {...getTableProps()}>
+      <TableHead>
+        {headerGroups.map(headerGroup => (
+          <TableRow {...headerGroup.getHeaderGroupProps()}>
+            {headerGroup.headers.map(column => (
+              <TableCell {...column.getHeaderProps()}>{column.render('Header')}</TableCell>
```

Comment:
minor: this feels less readable to me than the original formatting

Reply:
What was the original formatting?

Reply:
Like this:
```js
<TableCell {...column.getHeaderProps()}>
    {column.render('Header')}
</TableCell>
```

Reply:
reverted it